backend/app/services/execution/agents/research_agent.py

```
@@ -0,0 +1,351 @@
+            logger.info(
+                f"[{self.name}] Creating job: "
+                f"task_id={task_id}, subtask_id={subtask_id}"
+            )
+
+            try:
+                result = await gemini_client.create_interaction(
+                    input_text=input_text,
+                    agent=agent,
+                )
+            except GeminiInteractionError as e:
+                raise Exception(f"Failed to create deep research job: {e}") from e
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)